frontend/src/api/profile/profileApi.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
```

Comment:
This doesn't follow the pattern/architecture of the rest of the application - please follow it!